apps/wallet-dashboard/lib/utils/migration.ts

```diff
@@ -49,3 +46,97 @@ export function groupStardustObjectsByMigrationStatus(
 
     return { migratable, unmigratable };
 }
+
+export function getTotalTimelockedAmount(stardustOutputObjects: IotaObjectData[]): number {
+    let totalTimelockedAmount = 0;
+
+    for (const outputObject of stardustOutputObjects) {
+        const outputObjectFields = extractOutputFields(outputObject);
+
+        totalTimelockedAmount += parseInt(outputObjectFields.balance);
+
+        totalTimelockedAmount += parseInt(outputObjectFields.balance);
+        const depositReturnAmount = calculateStorageDepositReturn(outputObjectFields);
+
+        if (depositReturnAmount) {
+            totalTimelockedAmount += depositReturnAmount;
+        }
+    }
+
+    return totalTimelockedAmount;
+}
+
+interface MigratableObjectsData {
+    totalNativeTokens: number;
+    totalVisualAssets: number;
+    accumulatedIotaAmount: number;
+}
+
+export function summarizeMigrationValues({
```

Comment:
I have my doubts about this because when we look at the details, we'll need to summarize all the outputs again (both those who are migratable and those who are not)
I think the summarize should be generic for both types of outputs

Reply:
Update: We settled on  calculating values here, to show quickly on page load and then get the details for the categories "migrated / "unmigrated" in other function so we don't block the page load